pkg/metrics/metrics.go

```diff
@@ -30,31 +30,43 @@ const (
 	imdsOperationsErrorsCountName          = "imds_operations_errors_count"
 	imdsOperationsDurationName             = "imds_operations_duration_seconds"
 
-	// AdalTokenFromMSIOperationName ...
+	// AdalTokenFromMSIOperationName represents the duration of obtaining a token with MSI.
 	AdalTokenFromMSIOperationName = "adal_token_msi"
-	// AdalTokenFromMSIWithUserAssignedIDOperationName ...
+
+	// AdalTokenFromMSIWithUserAssignedIDOperationName represents the duration of obtaining a token with a user-assigned identity.
 	AdalTokenFromMSIWithUserAssignedIDOperationName = "adal_token_msi_userassignedid"
-	// AdalTokenOperationName ...
+
+	// AdalTokenOperationName represents the duration of obtaining a token.
 	AdalTokenOperationName = "adal_token"
-	// GetVmssOperationName ...
+
+	// GetVmssOperationName represents the duration of a GET request to a VMSS instance.
 	GetVmssOperationName = "vmss_get"
-	// PutVmssOperationName ...
+
+	// PutVmssOperationName represents the duration of a PUT request to a VMSS instance.
 	PutVmssOperationName = "vmss_create_or_update"
```

Comment:
Now that we have switched to using `PATCH`, I think we should rename this to `PatchVMSSOperationName = "vmss_update"`

Reply:
Changed to `UpdateVMSSOperationName = "vmss_update"` to unify variables name between metrics.go and stats.go.